config/klasses.yml

```diff
@@ -1,186 +1,222 @@
 klasses:
   Barbarian:
     Primary:
-      - Bash
-      - Cleave
-      - Frenzy
+      Bash                     : ["", "Frostbite", "Onslaught", "Punish", "Instigation", "Pulverize"]
```

Comment:
@rylnd this is due to "no rune" being an option, there was a reason for using an empty string in the initial implementation. I don't remember what it was.